app/controllers/orders_controller.rb

```diff
@@ -6,6 +6,9 @@ def index
   def create
     if session[:user_id].nil?
       redirect_to '/login'
+    elsif session[:cart].nil? || session[:cart].empty?
+      redirect_to '/cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.